### ISSUE TITLE
Server crashes when the buffer is freed during Synth execution

### ISSUE THREAD

**Issue by geoffroymontel** [elgiano/XPlayBuf#4]
Hello

Sorry to bother you again !
I noticed the server crashes when you free a buffer that is being played.

Small reproducer

```supercollider
b = Buffer.read(s, Platform.resourceDir+/+"sounds"+/+"a11wlk01.wav");

(
SynthDef(\xplaybuf, {
	var snd;
	snd = XPlayBuf.ar(1, \bufnum.kr(-1), \rate.kr(1), \trigger.tr(1), \startPos.kr(0));
	Out.ar(\out.kr(0), snd);
}).add;
)
(
{
	x = Synth(\xplaybuf, [bufnum: b]);
	1.wait;
	b.free;
}.fork
)
```

gives

```
Server 'localhost' exited with exit code -1073741819.
server 'localhost' disconnected shared memory interface
```

Tested on SC 3.12.1 on Windows 10. The server sample rate is 96 khz.

Best

Geoffroy

**Comment by elgiano** [comment]
fixed in v0.1.1! Thanks a lot for finding this!

**Comment by geoffroymontel** [comment]
Thank a lot Gianluca !
